larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc

```diff
@@ -95,7 +126,17 @@ const pandora::Cluster * const pCluster)
     float dTdLWidth(-1.f), straightLineLengthLarge(-1.f), diffWithStraightLineMean(-1.f), diffWithStraightLineSigma(-1.f), maxFitGapLength(-1.f), rmsSlidingLinearFit(-1.f);
     this->CalculateVariablesSlidingLinearFit(pCluster, straightLineLengthLarge, diffWithStraightLineMean, diffWithStraightLineSigma, dTdLWidth, maxFitGapLength, rmsSlidingLinearFit);
 
-    featureVector.push_back(straightLineLengthLarge);
+    if (m_ratioVariables && (straightLineLengthLarge > std::numeric_limits<double>::epsilon()))
+    {
+        diffWithStraightLineMean  /= straightLineLengthLarge;
+        diffWithStraightLineSigma /= straightLineLengthLarge;
+        dTdLWidth                 /= straightLineLengthLarge;
+        maxFitGapLength           /= straightLineLengthLarge;
+        rmsSlidingLinearFit       /= straightLineLengthLarge;
+    }
+
+    if(m_addStraightLineLength)
```

Comment:
Prefer "if (...)" - i.e. space between "f" and "("

Reply:
its the only one missing!